sdk/identity/identity/src/msal/utils.browser.ts

```diff
@@ -0,0 +1,293 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
The great majority of this module doesn't seem to be browser specific. Is it mostly duplicated code from the node version?

Reply:
Yes, we know of this issue as it's a quick fix for a big issue we have to address in the 4.0 timeframe.  This pattern is already in the 4.0 release as well.